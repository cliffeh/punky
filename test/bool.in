#t                                    ; #t
#f                                    ; #f
(not #f)                              ; #t
(not #t)                              ; #f
(not (= 5 7))                         ; #t
(and #f #f)                           ; #f
(and #t #f)                           ; #f
(and #f #t)                           ; #f
(and #t #t)                           ; #t
(and (< 6 7) (> 4 5))                 ; #f
(or #f #f)                            ; #f
(or #t #f)                            ; #t
(or #f #t)                            ; #t
(or #t #t)                            ; #t
(xor)                                 ; #f
(xor #t)                              ; #t
(xor #f)                              ; #f
(xor #f #f)                           ; #f
(xor #f #t)                           ; #t
(xor #t #f)                           ; #t
(xor #t #t)                           ; #f
(xor #f #f #f)                        ; #f
(xor #f #t #f)                        ; #t
(xor #t #t #t)                        ; #t
