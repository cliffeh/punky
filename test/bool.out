#t
#f
#t
#f
#t
#f
#f
#f
#t
#f
#f
#t
#t
#t
#f
#t
#f
#f
#t
#t
#f
#f
#t
#t
